lib/sourceParams.js

```diff
@@ -35,13 +35,13 @@ module.exports = function (app) {
 
   // default env to production
   process.env.NODE_ENV = 'production'
+  app.set('env', 'production')
 
   pkg.rooseveltConfig = pkg.rooseveltConfig || {}
 
   app.set('appDir', appDir)
   app.set('package', pkg)
   app.set('appName', pkg.name || 'Roosevelt Express')
-  app.set('appVersion', pkg.version)
```

Comment:
Removing this breaks a test in cssTest.js due to the fact that when you try to get the version code in the various preprocessors, they still depend on the appVersion field to be set.